aimmo-game/simulation/direction.py

```diff
@@ -16,6 +16,13 @@ def dict(self):
     def __repr__(self):
         return 'Direction(x={}, y={})'.format(self.x, self.y)
 
+    @staticmethod
+    def copy(other):
```

Comment:
they aren't actually, though they probably should be. Method name unclear in any case.